index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please use the [Microverse template](https://github.com/microverseinc/readme-template) to document your repo, I notice that the main branch already has it but it's not much descriptive and professional, so Kindly suggest adding The Microverse  README to your project.